packages/langium/src/services.ts

```diff
@@ -26,6 +26,7 @@ export type LangiumGeneratedServices = {
     Grammar: Grammar
     AstReflection: AstReflection
     LanguageMetaData: LanguageMetaData
+    ParserConfig: ParserConfig
```

Comment:
Please move this to the `parser` group:
```suggestion
    parser: {
        ParserConfig: ParserConfig
    }
```